docs/developer/developing.md

```diff
@@ -0,0 +1,49 @@
+# Developing Cincinnati
+
+## Overview
+
+The following sections describe how to build and test the project for local development. 
+
+
+## Prerequisites
+
+* Git
+* Rust
+* [just](https://github.com/casey/just)
+
+*Note:* We recommend using [Rustup](https://github.com/rust-lang/rustup/blob/master/README.md) for Rust toolchain management. 
+
+## Running it locally
+
+Please below commands in different terminal windows 
+```
+$ just run-graph-bulider
+```
+
+```
+$ just run-policy-engine
+```
+
+Here is the command to get graph for "stable-4.2" and for "amd64" architecture
+
+```
+just get-graph-pe "stable-4.2" "amd64"
```

Comment:
Earlier examples all included an explicit `$` prompt.  I like prompts, in which case we should have one here.  But it would also be internally consistent to remove all of the prompts and use:

    ```sh

Reply:
nit: no need to quote stable-4.2 or amd64.  Neither contains shell-sensitive characters.